### DISCUSSION TITLE
Sport Widget

### DISCUSSION THREAD

**Discussion by crlshn** [Lissy93/dashy#591] [Q&A]
Hi all, 

I notice the sport widget is giving the time ok (I whish localtime instead) but date wrong
the SportsDB shows on the website:
![image](https://user-images.githubusercontent.com/2448271/162038303-25f84aa5-6ce5-46e1-8222-7ea123d973fb.png)
but my Dashy shows wrong date: 1 day exactly off
![image](https://user-images.githubusercontent.com/2448271/162038428-3289bf81-1c53-4973-abab-368fc0a48246.png)

the docker container@synology is also wrong
![image](https://user-images.githubusercontent.com/2448271/162038718-4b2ab6b1-df92-416f-b89d-c046315fd4b9.png)

I would like directions on how correct the date and time on the dashy dashboard and maybe the ability to use localregional settings from the browser (mozilla or chrome) for the display on the date/time.

thanks in advance.
the screenshoots were taken around 
![image](https://user-images.githubusercontent.com/2448271/162039441-2273f592-3734-46a6-8c49-d429eef254df.png)

**Comment by lissy93** [comment]
So I'll look into this when I next get a moment, but in the meantime the code for that widget is in [`SportsScores.vue`](https://github.com/Lissy93/dashy/blob/master/src/components/Widgets/SportsScores.vue), and in the `processData()` method is where date is set. If the sports api doesn't support different time zones, then it'd need to be done in that function. We could do something similar to clock widget to get the users time zone, then convert the date accordingly.

**Comment by crlshn** [comment]
thank a lot,
I don't know vue, but momentJS could be usefull

```
import moment from 'moment'


Vue.filter('formatDate', function(value) {
  if (value) {
    return moment(String(value)).format('MM/DD/YYYY hh:mm')
  }
}

<span>{{yourDateString | formatDate}}</span>

```
